tests/behat/src/FixtureContext.php

```diff
@@ -508,4 +509,31 @@ public function iScrollTheEditorDetailsPanelToTheTop()
         $script = "document.querySelector('.editor__details fieldset').scrollTo(0, 0);";
         $this->getMainContext()->getSession()->executeScript($script);
     }
+
+    /**
+     * Example: Given a maximum file size is 5k
+     *
+     * @Given /^a maximum file size is "([^"]+)"$/
+     * @param string $size Max file size
+     */
+    public function stepCreateMaximumFileSizeStep($size): void
+    {
+        $uploadValidatorClass = Upload_Validator::class;
+        $config = <<<YAML
+---
+name: fileallowedsize
+---
+
+$uploadValidatorClass:
+    default_max_file_size:
+        '*': $size
+YAML;
```

Comment:
```suggestion
        ---
        name: fileallowedsize
        ---
        $uploadValidatorClass:
            default_max_file_size:
                '*': $size
        YAML;
```
Heredocs and nowdocs don't need to be aligned to the very left side anymore - we can keep them indented with the rest of the code.